05_syntax_lab/01.py

```diff
@@ -1,5 +1,18 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+"""
+Elad Agam
+01 - Get 10 numbers and return Max
+"""
+
+#init max
+print "Enter a Number"
+input = raw_input()
+Max = input
+
+for num in range(9):
+	print "Enter another Number"
+	input = raw_input()
+	if input > Max:
+		Max=input
+
+print "Max = {maximum}".format(maximum=Max)
+

```

Comment:
Done

On 4 July 2016 at 13:53, Gabor Szabo <notifications@github.com> wrote:

> In 05_syntax_lab/01.py
> <https://github.com/ynonp/python-examples-verint-2016-07/pull/5#discussion_r69441179>
> :
>
> > +01 - Get 10 numbers and return Max
> > +"""
> > +
> > +#init max
> > +print "Enter a Number"
> > +input = raw_input()
> > +Max = input
> > +
> > +for num in range(9):
> > +	print "Enter another Number"
> > +	input = raw_input()
> > +	if input > Max:
> > +		Max=input
> > +
> > +print "Max = {maximum}".format(maximum=Max)
> > +
>
> Hi Elad,
> thanks for your first solution.
> Your solution seems to work, but there can be a number of improvements.
> Please try them:
>
>    - Variable names are usually all lowercase, but max is also a keyword
>    in Python. Could you use a different variable name?
>    - Could you eliminate the duplicate call to raw_input?
>    - The call to format with the name is nice very nice!
>    - The "Enter another Number" now prints a newline before the user can
>    enter the input. Could you get rid of that newlines?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ynonp/python-examples-verint-2016-07/pull/5/files/375b8df6e995510bd3052ee7976317672deab854#r69441179>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ARMSUPYw-YZin-Pl_dKmss8AMT1vLitqks5qSOYagaJpZM4JEPzj>
> .
>